externs/tui-editor.externs.js

```diff
@@ -14,16 +14,15 @@ toastui.EventManager = function() {};
 
 /**
  * @param {string} eventName
- * @param {function()} cb
```

Comment:
It would not build with the parameter as just `function()` so I simplified it to just `*`. Looking at it again, I changed it to `function(string)` to better define what is being passed in, and it builds again.